### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by victyr**
Use basic JavaScript syntax.
Use JavaScript to manipulate DOM elements.
Use JavaScript events.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,8 +41,4 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
 ## 📝 License
```
Comment (zieeco) [inline README.md]: **[REQUIRED CHANGES]**

- Please, kindly add a license file.
- Please, kindly update the author header by adding the names of the author that has collaborated on this project.
- Please, kindly remove the headers that are not in use to make your readme more professional.
- Update the getting started headers by telling the user how to have your project on their local machine.
- Kindly update the acknowledgment header by acknowledging microverse.
- Kindly follow the [microverse readme template](https://github.com/microverseinc/readme-template) to restructure your readme so it conforms to the template format arrangement.

File: script.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const containMenu = document.getElementById('container_menu');
+const hamburgerMenu = document.getElementById('hamburger_menu');
+const closeButton = document.getElementById('close_button');
+const navElement = document.querySelectorAll('.menu_mobile a');
+
```
Comment (zieeco) [inline script.js:6]: **[REQUIRED CHANGES]**

- Please, kindly ensure when any of the mobile menu links are clicked, the mobile menu disappears.
*Hint: Check the class `.menu_mobile`*, I don't think that class exists in the index.html file. 
see the video below:

https://user-images.githubusercontent.com/59397159/173724765-6e40be3f-1fa1-46b9-8763-34cfa12ab748.mp4

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good commit messages. ✔️



## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -4,180 +4,228 @@
...
-        <a href="#" class="toolbar logo"><img src="images/logo.png" alt="" /></a>
-        <label>&#9776;</label>
-      </div>
-      <div class="desk-nav">
-        <a href="#">Portfolio</a>
-        <a href="#">About</a>
-        <a href="#">Contact</a>
-      </div>
-    </header>
-    <main class="main-sec">
```
Comment (koachcode) [inline index.html:26]: - Please just as the previous reviewer requested changes on the mobile menu, please implement the changes. 
  - On the mobile menu when the user clicks on the hamburger menu it shows the menu, but there is no `X` to close the menu. 
  - The menu design does not look like that of the Figma design, please fix it so it looks similar to the Figma design. 

##### Your design
Screenshot from 2022-06-15 12-37-01

##### Figma design
Screenshot from 2022-06-15 12-43-19

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @victyr , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

keep-trying-youre-almost-there

 ## Highlights
- No linters error :+1: 
- Good commit messages :+1: 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — COMMENTED** [review]
Hello @victyr  👋,

_This is an additional review as a result of the second opinion request._

We checked your project and couldn't see the same issue. we are sorry for the inconvenience.

**As this was the only reason that your project was not approved it will be approved as soon as possible.**

Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approve right away :)

Keep rocking 🚀!

File: script.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+const closeSidebar = document.querySelector('.sidebar_menu');
+const closeButton = document.getElementById('close_button');
+const navElement = document.querySelectorAll('.menu_mobile a');
+
+hamburgerMenu.addEventListener('click', () => {
+  containMenu.style.display = 'flex';
+});
+
+closeSidebar.addEventListener('click', () => {
+  containMenu.style.display = 'none';
```
Comment (chrystalme) [inline script.js:12]: Kindly keep your code `D-R-Y`. This same code is repeated 3 times. This defeats the purpose of `OOP`. You could use this code as a function and call it in the `addEventListener` method as the second argument

File: script.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+});
+
+closeButton.addEventListener('click', () => {
+  containMenu.style.display = 'none';
```
Comment (chrystalme) [inline script.js:16]: Kindly keep your code `D-R-Y`. This same code is repeated 3 times. This defeats the purpose of `OOP`. You could use this code as a function and call it in the `addEventListener` method as the second argument

File: script.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+});
+
+navElement.forEach((element) => {
+  element.addEventListener('click', () => {
+    containMenu.style.display = 'none';
```
Comment (chrystalme) [inline script.js:21]: Kindly keep your code `D-R-Y`. This same code is repeated 3 times. This defeats the purpose of `OOP`. You could use this code as a function and call it in the `addEventListener` method as the second argument

File: script.js (hunk 2 above)
Comment (chrystalme) [reply script.js:6]: As the previous reviewer requested, you `button` position is a little off. Kindly check your CSS and fix it.

**Review by chrystalme — CHANGES_REQUESTED** [review]
## Hi @victyr,

Great work so far on the project. 👏🏽 👏🏽 👏🏽  
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes ♻️ 
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA). Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by victyr** [comment]
@ginohmk kindly see the previous comments on the changes made . I have just
used my last pill request and all changes are in the link

**Comment by mke2111** [comment]
Project Under Second Opinion Review.
Kindly wait for a response. 😄

**Review by mke2111 — COMMENTED** [review]
Hello @victyr 👋,

_This is an additional review as a result of the second opinion request._

Great work on this milestone so far. 👍🏽 

The changes requested by @chrystalme should have been optional since the previous CRs had not mentioned them to you before. Please consider them optional, but I strongly suggest that you make these implementations since they all benefit your skills and your page.


As this was the only reason that your project was not approved it will be approved as soon as possible.
Please note that since we will approve it manually so this could take a few hours!

Keep rocking 🚀!

**Pull request merged**